src/data_vis.ts

```diff
@@ -122,12 +190,47 @@ class DataVisModel {
             var blob = new Blob([csv], {type: "text/plain;charset=utf-8"});
             saveAs(blob, "data.csv");
         });
-    }
+    };
 
     changeBurden(burden: string) {
         this.humanReadableBurdenOutcome(burden)
         this.plotTitle(this.defaultTitle());
-    }
+        this.onUIChange(true, true);
+    };
+
+    onUIChange(redraw: boolean, updateUI: boolean) {
```

Comment:
On the whole I think functions that change behaviour based on boolean arguments should be avoided - it's usually an indication that there are multiple different functions masquerading as one. In this case I'd split out the updating of options and the redrawing of the plots into two separate functions.